ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -555,6 +561,48 @@ func (ipfs *Connector) RepoStat() (api.IPFSRepoStat, error) {
 	return stats, nil
 }
 
+// Resolve accepts ipfs or ipns path and resolves it into a cid
+func (ipfs *Connector) Resolve(path string) (cid.Cid, error) {
+	validPath, err := gopath.ParsePath(path)
```

Comment:
this logic should be in `cluster`. This component is just a client to IPFS so Resolve should just call `resolve` on IPFS, not do the extra logic that may actually end up not calling resolve.

Reply:
I've thought about it a bit more.. you can leave this here, it's good.

Reply:
Moved already :)
If you want I can change it back, but I like the idea of keeping ipfs connector just a client to IPFS

Reply:
OK, leave it as you have it now and we'll check later.